packages/smaz-decompress/src/index.ts

```diff
@@ -26,3 +26,55 @@ export class SmazDecompress {
     return output;
   }
 }
+
+const EMPTY_UINT8_ARRAY = new Uint8Array(0);
+const TEXT_ENCODER = new TextEncoder();
```

Comment:
Could we keep the use of `TextEncoder` inline just in case this code is used in an environment without `TextEncoder` available as a global? I'd like to keep backward compatibility to a maximum with this library.